code/__DEFINES/mod.dm

```diff
@@ -36,5 +36,13 @@
 #define MOD_ICON_OVERRIDE "mod_icon_override"
 #define MOD_WORN_ICON_OVERRIDE "mod_worn_icon_override"
 
+//Defines for MODlink frequencies
+#define MODLINK_FREQ_NANOTRASEN "NT"
+#define MODLINK_FREQ_SYNDICATE "SYND"
+#define MODLINK_FREQ_CHARLIE "CHRL"
+#define MODLINK_FREQ_CENTCOM "CC"
+
 /// Global list of all /datum/mod_theme
 GLOBAL_LIST_INIT(mod_themes, setup_mod_themes())
```

Comment:
This shouldn't be statically initialized, please do this in the world/New called proc for setting up datum lists.

Reply:
```suggestion
GLOBAL_LIST_EMPTY(mod_themes)
```

Reply:
ok i guess